<launch>
	<node name="custom_message_talker" pkg="custom_msgs" type="custom_talker.py" output="screen"/>
</launch>